python-checks/src/test/resources/checks/verifiedSslTlsCertificateCheck.py

```diff
@@ -236,3 +236,12 @@ def call_expression_in_arguments():
     import ssl
     conn = httplib.client.HTTPSConnection("123.123.21.21", context=ssl._create_unverified_context()) # Noncompliant {{Enable server certificate validation on this SSL/TLS connection.}}
 #                                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^ 0
+
+def httpx_verify():
+    import httpx
+
+    httpx.get("https://expired.badssl.com/", verify=False)  # Noncompliant
+    httpx.get("https://expired.badssl.com/")  # Compliant
```

Comment:
Perhaps it's an overkill, but why don't we include test cases for all the other function names?